MAUI/Kanban-Board/Column.md

```diff
@@ -72,26 +71,6 @@ kanban.ColumnMappingPath = "ID";
 
 {% endtabs %}
 
-### Multiple category for a column
```

Comment:
@EzhilarasanElangovan31 update the content for single column -> Category for a column and update content and code snippet

Reply:
Updated the content